### PR TITLE
GIX-1985: Remove ENABLE_MY_TOKENS flag

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Remove `ENABLE_MY_TOKENS` flag.

# Changes

* Remove flag from `dfx.json`.
* Remove flag from environment constants.
* Remove flag from flags store.

# Tests

* Remove flag from vitests setup.
* Remove flag from config tests.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #4434** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4434?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4434?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd please review

File: scripts/nns-dapp/migration-test.getters.test
Comment (dskloetd) [inline scripts/nns-dapp/migration-test.getters.test:1]: Is this used by anything? It also has a bunch of old feature flags that don't exist anymore...

**Review by dskloetd — APPROVED** [review]

File: scripts/nns-dapp/migration-test.getters.test
Comment (lmuntaner) [reply scripts/nns-dapp/migration-test.getters.test:1]: I honestly don't know. I searched the flag and removed it.

**Pull request merged**